terraform/deployments/cluster-services/templates/alertmanager-config.tpl

```diff
@@ -39,6 +39,8 @@ alertmanager:
 
          *Environment:* ${environment}
 
+         {{ if contains(keys(.CommonAnnotations), "runbook") }}*Runbook:* {{ .CommonAnnotations.runbook }}{{ end }}
```

Comment:
Probably don't need to bother checking for the annotation, as we know the alert is going to have runbook
```suggestion
         *Runbook:* {{ .CommonAnnotations.runbook }}
```

Reply:
(Have added and rebased, because I also updated it to runbook_url)